projects/L4T/packages/glibc/package.mk

```diff
@@ -0,0 +1,168 @@
+PKG_SHORTDESC="glibc: The GNU C library"
+PKG_LONGDESC="The Glibc package contains the main C library. This library provides the basic routines for allocating memory, searching directories, opening and closing files, reading and writing files, string handling, pattern matching, arithmetic, and so on."
+PKG_BUILD_FLAGS="-gold -lto"
+
+PKG_CONFIGURE_OPTS_TARGET="BASH_SHELL=/bin/sh \
+                           ac_cv_path_PERL=no \
+                           ac_cv_prog_MAKEINFO= \
+                           --libexecdir=/usr/lib/glibc \
+                           --cache-file=config.cache \
+                           --disable-profile \
+                           --disable-sanity-checks \
+                           --enable-add-ons \
+                           --enable-bind-now \
+                           --with-elf \
+                           --with-tls \
+                           --with-__thread \
+                           --with-binutils=${BUILD}/toolchain/bin \
+                           --with-headers=${SYSROOT_PREFIX}/usr/include \
+                           --enable-kernel=3.0.0 \
+                           --without-cvs \
+                           --without-gd \
+                           --disable-obsolete-rpc \
+                           --disable-build-nscd \
+                           --disable-nscd \
+                           --enable-lock-elision \
+                           --disable-timezone-tools"
+
+if [ "${DEBUG}" = yes ]; then
+  PKG_CONFIGURE_OPTS_TARGET="${PKG_CONFIGURE_OPTS_TARGET} --enable-debug"
+else
+  PKG_CONFIGURE_OPTS_TARGET="${PKG_CONFIGURE_OPTS_TARGET} --disable-debug"
+fi
+
+NSS_CONF_DIR="${PKG_BUILD}/nss"
+
+GLIBC_EXCLUDE_BIN="catchsegv gencat getconf iconv iconvconfig"
+GLIBC_EXCLUDE_BIN="${GLIBC_EXCLUDE_BIN} makedb mtrace pcprofiledump"
+GLIBC_EXCLUDE_BIN="${GLIBC_EXCLUDE_BIN} pldd rpcgen sln sotruss sprof xtrace"
+
+pre_build_target() {
+  cd "${PKG_BUILD}"
+    aclocal --force --verbose
+    autoconf --force --verbose
+  cd -
+}
+
+pre_configure_target() {
+# Filter out some problematic *FLAGS
+  export CFLAGS=`echo "${CFLAGS}" | sed -e "s|-ffast-math||g"`
+  export CFLAGS=`echo "${CFLAGS}" | sed -e "s|-Ofast|-O2|g"`
+  export CFLAGS=`echo "${CFLAGS}" | sed -e "s|-O.|-O2|g"`
+  export CFLAGS=`echo "${CFLAGS}" | sed -e "s|-Werror||g"`
+  export CFLAGS=`echo "${CFLAGS}" | sed -e "s|-mthumb||g"`
+  if [ -n "${PROJECT_CFLAGS}" ]; then
+    export CFLAGS=`echo "${CFLAGS}" | sed -e "s|${PROJECT_CFLAGS}||g"`
+  fi
+
+  export LDFLAGS=`echo "${LDFLAGS}" | sed -e "s|-ffast-math||g"`
+  export LDFLAGS=`echo "${LDFLAGS}" | sed -e "s|-Ofast|-O2|g"`
+  export LDFLAGS=`echo "${LDFLAGS}" | sed -e "s|-O.|-O2|g"`
+  export LDFLAGS=`echo "${LDFLAGS}" | sed -e "s|-Wl,--as-needed||"`
+  export LDFLAGS=`echo "${LDFLAGS}" | sed -e "s|-Werror||g"`
+
+  unset LD_LIBRARY_PATH
+
+  # set some CFLAGS we need
+  export CFLAGS="${CFLAGS} -g -fno-stack-protector -Wno-error -Wmissing-attributes"
```

Comment:
use bash builtin substitution like VAR="${VAR//search/replace}" and then export only once